tests/run-make/incremental-debugger-visualizer/rmake.rs

```diff
@@ -0,0 +1,58 @@
+// This test ensures that changes to files referenced via //[debugger_visualizer]
```

Comment:
```suggestion
// This test ensures that changes to files referenced via #[debugger_visualizer]
```

search-and-replace error